webapp/src/hooks/hooks.js

```diff
@@ -10,6 +11,14 @@ export default class Hooks {
 
     slashCommandWillBePostedHook = (message, contextArgs) => {
         if (message && (message.startsWith('/jira create ') || message === '/jira create')) {
+            if (!isInstanceInstalled(this.store.getState())) {
+                sendEphemeralPost(this.store, 'There is no Jira instance installed. Please contact your system administrator.');
+                return Promise.resolve({});
+            }
+            if (!isUserConnected(this.store.getState())) {
+                sendEphemeralPost(this.store, 'Your username is not connected to Jira. Please type `/jira connect`.');
```

Comment:
For text, propose

```
Your Mattermost account is not connected to Jira. Please use `/jira connect` to connect your account, then try again.
```

Reply:
@jasonblais Yes, but also for the case where there is no instance installed.

This only really applies to the `/jira create` case, because it has to be handled on the client side. All the other slash commands are handled on the server, and Lev's refactoring will include easy-to-apply permissions filters for all commands. But sure, we should open tickets to make sure they're in place after the refactor.

Reply:
> This only really applies to the /jira create case, because it has to be handled on the client side. All the other slash commands are handled on the server, and Lev's refactoring will include easy-to-apply permissions filters for all commands. But sure, we should open tickets to make sure they're in place after the refactor.

Okay, good to know. This is fine to special case for `/jira create` in that case and consider improving the server-side work later if/as needed 👍